web_tree_date_search/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Tree Date Search
+==============================
```

Comment:
Same length for the title and the line. Put also a line above. Check latest README template here: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst